client/app/assets/sprites/icon-activity.svg

```diff
@@ -0,0 +1,16 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48">
+  <g fill="none" fill-rule="evenodd" transform="translate(-96 -96)">
+    <g stroke-width="2" transform="translate(97 97)" stroke-linecap="round" stroke-linejoin="round">
+      <g stroke="#95999D" transform="translate(0 16)">
+        <path d="M0.0254,30 L46.0254,30"/>
+        <polygon points="8.026 20 2.026 20 2.026 30 8.026 30"/>
+        <polygon points="20.026 10 14.026 10 14.026 30 20.026 30"/>
+        <polygon points="32.026 14 26.026 14 26.026 30 32.026 30"/>
+        <polygon points="44.026 0 38.026 0 38.026 30 44.026 30"/>
+      </g>
+      <g stroke="currentColor" transform="translate(3)">
```

Comment:
Interesting, I didn't know that we could use the CSS `color` property here using `currentColor`. Good to know 👍